components/Footer.js

```diff
@@ -6,32 +6,48 @@ import { trackEvent } from './Analytics'
 import { PLATFORM_NAME, LOGO_FOOTER, TEAM_EMAIL, INSTAGRAM_URL, FACEBOOK_URL, TWITTER_URL, NEWSLETTER, DISCORD_URL } from '../config';
 
 const footer = () => (
-  <div className="footer-wrapper no-print bg-primary">
+  <div className="bg-slate-50  border-t border-t-neutral-700">
     { NEWSLETTER && <Newsletter placement="Footer" /> }
-    <footer className="main-content">
-      <div className="flex flex-row items-center justify-between">
-        <div className="flex flex-row items-center justify-start">
+    <footer className="flex flex-col items-center p-4">
+        {/* <div className="flex flex-row items-center justify-start">
           { LOGO_FOOTER && <img src={LOGO_FOOTER} width="110" alt={ PLATFORM_NAME } /> }
-        </div>
-        <div className="py-10 px-2">
-          <div className="flex flex-row items-center justify-between mb-2">
+        </div> */}
+        <div className="flex flex-col md:flex-row py-2 items-center w-11/12 justify-between">
+          <div className='flex flex-col items-center'>
+          <div className="flex flex-row space-x-7 self-center p-4 mb-8 md:mb-1">
             { INSTAGRAM_URL && <a href={ INSTAGRAM_URL } target="_blank" rel="noreferrer nofollow">
-              <img src="/images/icons/instagram.svg" width="30" alt="instagram" />
+              <img src="/images/icons/instagram.svg" width="40" alt="instagram" className="bg-primary rounded-3xl p-1"/>
             </a> }
             {FACEBOOK_URL && <a href={ FACEBOOK_URL } target="_blank" rel="noreferrer nofollow">
-              <img src="/images/icons/facebook.svg" width="30" alt="facebook" />
+              <img src="/images/icons/facebook.svg" width="40" alt="facebook" className="bg-primary rounded-3xl p-1" />
             </a> }
             { TWITTER_URL && <a href={ TWITTER_URL } target="_blank" rel="noreferrer nofollow">
-              <img src="/images/icons/twitter.svg" width="30" alt="twitter" />
+              <img src="/images/icons/twitter.svg" width="40" alt="twitter" className="bg-primary rounded-3xl p-1" />
             </a> }
             { DISCORD_URL && <a href={ DISCORD_URL } target="_blank" rel="noreferrer nofollow">
-              <img src="/images/icons/discord.svg" width="30" alt="twitter" />
+              <img src="/images/icons/discord.svg" width="40" alt="twitter" className="bg-primary rounded-3xl p-1"/>
             </a> }
           </div>
-          <div className="border-t border-background pt-2">
+          <p className='m-2 p-2 hidden md:flex'>Powered by: closer</p>
+          </div>
+          {/* <div className="border-t border-background pt-2">
             <a href={`mailto:${TEAM_EMAIL}`} className="text-sm text-white hover:underline">{TEAM_EMAIL}</a>
+          </div> */}
+          
+          <div className='flex flex-col items-start text-sm'>
```

Comment:
There is a <Newsletter /> component (line 10) - please use that, and can update the styles to match.